### ISSUE TITLE
About the new code

### ISSUE THREAD

**Issue by LeeX-Bruce** [YazhouZhu19/RPT#12]
Dear author, 

I noticed that your recently updated code has some parameters that are different from those of the paper, such as `self.fg_num = 10 # number of foreground partitions` to `self.fg_num = 100 # number of foreground partitions`. And the loss function has changed too. 

Could you please tell me why there are these changes. Thanks!

**Comment by YazhouZhu19** [comment]
`self.fg_num = 100` may lead to the better performance in some datasets. It depends on which datasets you chosen, and it also can be change according to your local conditions. 

May these can help you, :)

**Comment by LeeX-Bruce** [comment]
Hi author, thank you for your previous reply.

Now I found a problem when testing the model. When testing, `ALL_EV = (1)` and `ALL_SUPP = (3)`, the Test Class is **SPLEEN** and it has a very low Dice score, only about **8%** , is this normal or is there something wrong with my image data.

Also, the model I got from my own training still gives poor results when tested, can you say more about what to do with the data used for training, all I need to do is to download the following data directly and do I need to do additional processing?
![image](https://github.com/YazhouZhu19/RPT/assets/56881416/d4772476-8cb0-4e3f-9678-ebd47f4b71d8)

Looking forward to your reply!

**Comment by YazhouZhu19** [comment]
`ALL_EV = (1)` means you just test the second fold for evaluation, this work employs the five-fold cross validation strategy, and the final result is the mean of five folds' results. `ALL_SUPP` please keep equal to 2 as used in the raw script. 

All of data we uploaded and the supervoxels have already been processed, there are no additional operations needed to be conducted. Due to no accessible to your local code files and environments, I can't understand why you obtain the poor results when tested. Can you provide your codes file and the test results? And how about the test results of my uploaded checkpoints' results under the raw scripts' settings?
